### PR TITLE
Feature/improve interface

### PR DESCRIPTION
**Pull request by HarwinBorger**

### CODE REVIEW

File: scss/modules/_image-focus.scss [hunk 1]
```diff
@@ -26,6 +20,12 @@
 
 		width: 100%;
 		height: 100%;
+
+		transition: 800ms all;
```
Comment (DannyvanHolten) [inline scss/modules/_image-focus.scss]: curves gebruiken?

 ik vind eigenlijk dat de notatie in secondes moet. .8s in dit geval, maar ik ben benieuwd of daar misschien artikelen over te vinden zijn welke notatie het beste is.

File: scss/modules/_image-focus.scss [hunk 2]
```diff
@@ -26,6 +20,12 @@
...
+
+		&:hover {
+			background-color: rgba(0, 0, 0, .1);
```
Comment (DannyvanHolten) [inline scss/modules/_image-focus.scss]: `background-color: rgba(#000, .1);`

File: scss/modules/_image-focus.scss [hunk 3]
```diff
@@ -69,9 +60,41 @@
 			height: 10px;
 
 			border: 2px solid #fff;
+			background-color: rgba(0, 0, 0, .5);
```
Comment (DannyvanHolten) [inline scss/modules/_image-focus.scss]: `background-color: rgba(#000, .5);`

File: scss/modules/_image-focus.scss [hunk 4]
```diff
@@ -69,9 +60,41 @@
...
 			border-radius: 50%;
 
 			transform: translate(-50%, -50%);
+
+			transition: 200ms all;
```
Comment (DannyvanHolten) [inline scss/modules/_image-focus.scss]: .2s heeft voorkeur wat mij betreft

File: scss/modules/_image-focus.scss [hunk 5]
```diff
@@ -69,9 +60,41 @@
...
+	&.is-hover {
+		.image-focus {
+			&__point {
+				&::before {
+					background-color: #008ec2;
+				}
+			}
+
+			&__clickarea {
+				background: rgba(0, 0, 0, .2); //Transparent layer
```
Comment (DannyvanHolten) [inline scss/modules/_image-focus.scss]: Alleen background-color aangeven als je niet de hele background hoeft aan te geven.

`background-color: rgba(#000, .2);`

File: scss/modules/_image-focus.scss [hunk 6]
```diff
@@ -69,9 +60,41 @@
...
+	&.is-active {
+		.image-focus {
+			&__point {
+				&::before {
+					background-color: #f90;
+				}
+			}
+
+			&__clickarea {
+				background: rgba(0, 0, 0, .4); //Transparent layer
```
Comment (DannyvanHolten) [inline scss/modules/_image-focus.scss]: `background-color: rgba(#000, .4);`

**Comment by HarwinBorger** [comment]
I resolved all comments. However the css transition with curve has some performance issues on the big black translucent clickarea so I didn't fully apply that improvement

**Comment by DannyvanHolten** [comment]
You did not answer or edit the question about seconds vs miliseconds. I found that w3schools uses seconds in all of their examples: https://www.w3schools.com/cssref/css3_pr_transition-duration.asp

**Comment by HarwinBorger** [comment]
> You did not answer or edit the question about seconds vs miliseconds.

As w3schools refers to 'seconds' there will be many examples were they use 'ms'. 
 w3.org says in the following article: ( https://www.w3.org/TR/css3-values/#time )

> All <time> units are compatible, and s is their canonical unit.

Which declares still nothing. They just convert all time units to a global one. 

As far as I can found it is preference what let you choose between 'seconds' and 'miliseconds'. Which can be chosen like what you would pronounce in the real world. Are you talking about:
- 0.15m or 15cm (in this case 15cm is  probably more logic)
- 2 meter or 200 cm (in this case (2m is probably more logic)

I would say everything below 1s is written in milliseconds, this since you're working on a scale of 0 till 1s. Everything above 1s is written in seconds since you're working on a scale of 1s and more (less precise)

**Comment by DannyvanHolten** [comment]
I do not agree, we are working in .5s and 1.5s, the difference is not as big as you say it is. I'd rather keep the same unit for everything.

**Pull request merged**